schemas/metadata/error/error.1.schema.json

```diff
@@ -6,6 +6,7 @@
       "type": "string"
     },
     "client_id": {
+      "pattern": "^[a-fA-F0-9]{8}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{4}-[a-fA-F0-9]{12}$",
```

Comment:
r? @jklukas

Reply:
For the error schema, it seems wrong to imply that values must follow this pattern. Imagine if a record comes to the decoder and fails validation _because_ the client_id was in an incompatible format. Currently, I believe the validation failure would cause the message to go to error output before the client_id is added as an attribute, but I could imagine wanting to try to get the errant client_id into attributes for the error message to help with debugging.

At the least, we'd expect client_id to not be populated in many cases for error messages.

Reply:
oooh, good point, will remove